regression/goto-analyzer/dependence-graph10/test.desc

```diff
@@ -0,0 +1,25 @@
+CORE
+main.c
+--dependence-graph --show
+activate-multi-line-match
+EXIT=0
+SIGNAL=0
+// First assignment has a control dependency on the if
+\/\/ ([0-9]+).*\n.*IF.*i < 10.*THEN(.*\n)*Control dependencies: \1\n(.*\n){2,3}.*a = 1
+// Second assignment has a control dependency on the if
+\/\/ ([0-9]+).*\n.*IF.*i < 10.*THEN(.*\n)*Control dependencies: \1\n(.*\n){2,3}.*a = 2
```

Comment:
There's only one `IF i < 10` how come it appears here twice with both dependencies?

Reply:
That's because `a = 1` and `a = 2` appear in the then and else branches of this if. So the statements in both branches have a control dependency on the goto that decides which branch to take.